std_msgs/Header header
float64[] tLast
geometry_msgs/Point[] poseArray
geometry_msgs/Quaternion[] orientationArray